docs/_documentations/remotedeploy-eclipse.md

```diff
@@ -56,10 +58,10 @@ This example uses Docker Hub:
 
 ![New Registry](./images/remoteeclipse/newReg1.png)
 
-- Add your Docker Hub user name.
+- Add your Docker Hub username.
 - Add your Docker Hub password.
```

Comment:
Should we specifically call out Docker Hub here or should we use just a more generic term like "container registry" which would also include standalone registries or other 3rd party services like Quay.io, Amazon ECR, Azure Container registry etc.

Reply:
Have substituted the more generic term "container registry" for Docker Hub.